### PR TITLE
service: change to system directories

### PR DESCRIPTION
**Pull request by rmsyn**
Change directories in the example systemd service file to system directories.

### CODE REVIEW

File: evremap.service [hunk 1]
```diff
@@ -3,7 +3,7 @@ WorkingDirectory=/
 # For reasons I don't care to troubleshoot, Fedora 31 won't let me start this
 # unless I use `bash -c` around it.  Putting the command line in directly
 # yields a 203 permission denied error with no logs about what it didn't like.
-ExecStart=bash -c "/home/wez/github/evremap/target/release/evremap remap /home/wez/github/evremap/pixelbookgo.toml"
+ExecStart=bash -c "/usr/bin/evremap remap /etc/evremap/remap.toml"
```
Comment (wez) [inline evremap.service:6]: I'm not sure that we need a new directory for a simple single file config, so I would lean towards this:

```suggestion
ExecStart=bash -c "/usr/bin/evremap remap /etc/evremap.toml"
```

**Review by wez — COMMENTED** [review]
Thanks!

File: evremap.service (hunk 1 above)
Comment (rmsyn) [reply evremap.service:6]: I also ran into an issue with the `WantedBy` target (no GDM). Are you OK with changing it to a generic target like `multi-user.target` or similar?

Comment (wez) [reply evremap.service]: Yeah, go for it!

**Comment by wez** [comment]
Thanks!

**Pull request merged**